xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxCC.h

```diff
@@ -31,7 +31,7 @@ class CDVDDemuxCC : public CDVDDemux
   explicit CDVDDemuxCC(AVCodecID codec);
   ~CDVDDemuxCC() override;
 
-  void Reset() override {};
+  bool Reset() override {};
```

Comment:
Having trouble with this not setting return value in Win64

Reply:
add return true;